src/controllers/import.js

```diff
@@ -138,7 +138,14 @@ function ImportController(context) {
 
     try {
       validateImportApiKey(importApiKey);
-      const job = await importSupervisor.getImportJob(jobId, importApiKey);
+      let job = await importSupervisor.getImportJob(jobId, importApiKey);
+      // TODO: Modify importApiKey to hashedKey
```

Comment:
Once this PR : https://github.com/adobe/spacecat-shared/pull/312 is merged in, I'll modify the importApiKey to a hashedKey.